src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupsRestClient.java

```diff
@@ -147,8 +147,7 @@ public QueryRequest query() {
     protected List<GroupInfo> query(QueryRequest queryRequest) throws RestApiException {
```

Comment:
Could you please add a comment above the method declaration saying that this method does not support Gerrit versions lower than 3.2.0? 

Everything else looks good to me. Thank you for taking care of it!